libcodechecker/report.py

```diff
@@ -151,6 +157,96 @@ def compare_ctrl_sections(curr, prev):
         return ''
 
 
+def remove_whitespace(source_line, col):
```

Comment:
@gyorb Last year I created a [pull request](https://github.com/Ericsson/codechecker/pull/1291/files#diff-3a366c56fb290a6aab1a67cfed0ab78bR412) where I created a `get_new_line_col_without_whitespace` function which did the same thing as this function but was more simpe than this. Could you please check it?

Reply:
I've updated the code to work as the previous version, all whitespaces are removed from the source line.